app/routes/_index/index.module.scss

```diff
@@ -38,3 +38,7 @@
     font-size: clamp(60px, 6vw, 110px);
     text-transform: uppercase;
 }
+
+.featuredProducts {
```

Comment:
Um. Both sections are featured products, but only uses this class name.

Rename to `.alternateBackground`

Reply:
It can also be moved to `common.scss`